File: src/gatling/scala/com/example/Runner.scala
package com.example

import com.example.simulations.{HelloWorldSimulation, ScalaLoadSimulation}
import io.gatling.app.Gatling
import io.gatling.core.config.GatlingPropertiesBuilder

object Runner {

  def main(args: Array[String]): Unit = {

    val simulationClass = classOf[ScalaLoadSimulation].getName

    val props = new GatlingPropertiesBuilder
    props.simulationClass(simulationClass)

    Gatling.fromMap(props.build)

  }

}

File: src/gatling/kotlin/com/example/Runner.kt
package com.example


import com.example.simulations.KotlinLoadSimulation
import io.gatling.app.Gatling
import io.gatling.core.config.GatlingPropertiesBuilder

fun main() {
    val props = GatlingPropertiesBuilder()
    props.simulationClass(KotlinLoadSimulation::class.java.name)

    Gatling.fromMap(props.build())
}